docs/architecture/mobile-clients/watchOS.md

```diff
@@ -21,8 +21,8 @@ Then in the Extension we have a layered architecture:
 
 ## Integration with iOS
 
-The watchOS app is developed using `XCode` and `Swift` and we need to integrate it to the `Xamarin`
-iOS application.
+The watchOS app is developed using `XCode` and `Swift` and we need to integrate it to the `MAUI` iOS
```

Comment:
We use code formatting for MAUI here, but leave it without in the overview.

Reply:
I can remove the code formatting here as well